ceaos/sensors/nwsensor.py

```diff
@@ -0,0 +1,45 @@
+from .sensor_definition import Sensor
+from ..loggers.InfluxDB import InfluxDBLogger
+
+
+class NetworkSensor(Sensor):
+    def __init__(self, logger=None, datatype=None, location=None):
+        self.curr_value = None
+        self.logger = logger
+        self.datatype = datatype
+        self.location = location
+        self.calibrate = None
+        self.refresh = None
+
+    def read_value(self):
+        return self.curr_value
+
+    def calibrate(self):
```

Comment:
This could cause an issue with there being a variable with the same name. Do we need a method here? What would calibrating a network sensor mean?